package.json

```diff
@@ -17,7 +17,7 @@
     "start:covid": "openmrs develop --backend https://ohri-dev.globalhealthapp.net --sources packages/esm-covid-app",
     "start:hiv": "openmrs develop --backend https://ohri-dev.globalhealthapp.net --sources packages/esm-hiv-app",
     "start:cervical-cancer": "openmrs develop --backend https://ohri-dev.globalhealthapp.net --sources packages/esm-cervical-cancer-app",
-    "start:pmtct": "openmrs develop --backend https://ohri-dev.globalhealthapp.net --sources packages/esm-ohri-pmtct-app",
```

Comment:
@kajambiya Should we leave this here?

Reply:
We can may be for uniformity but at the end of the day we shall never run pmtct against ohri dev since those concepts are not in ohri dev